flask_admin/contrib/sqla/filters.py

```diff
@@ -118,6 +120,38 @@ def operation(self):
         return lazy_gettext('not in list')
 
 
+# Composite filters
+class FilterAndEqual(FilterEqual):
```

Comment:
This filter is essentially the same as adding two filters from the user interface? If that's the case, using the two seperate filters seems like it would be much easier to understand.